apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -1287,22 +1260,6 @@ settingsRegistry.addGroup('Message', function () {
 		actionText: 'clear',
 		i18nLabel: 'clear_cache_now',
 	});
-	// TODO: deprecate this setting in favor of App
-	this.add('API_EmbedDisabledFor', '', {
-		type: 'string',
-		public: true,
-		i18nDescription: 'API_EmbedDisabledFor_Description',
-		alert: 'This_is_a_deprecated_feature_alert',
-	});
-	// TODO: deprecate this setting in favor of App
-	this.add('API_EmbedIgnoredHosts', 'localhost, 127.0.0.1, 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16', {
-		type: 'string',
-		i18nDescription: 'API_EmbedIgnoredHosts_Description',
-	});
-	// TODO: deprecate this setting in favor of App
-	this.add('API_EmbedSafePorts', '80, 443', {
-		type: 'string',
-	});
```

Comment:
Are you sure we should remove those settings? They are important to prevent localhost lookup

Reply:
Added again